### DISCUSSION TITLE
Ocarina of time dungeon crash

### DISCUSSION THREAD

**Discussion by gamegirlniecy** [mupen64plus-ae/mupen64plus-ae#894] [Q&A]
Please help. I am on the spirit temple and everytime i go through the first door after the beak puzzle the game crashes and takes me back to the mupen64plus main menu.

**Comment by fzurita** [comment]
First, are you using this app?

https://play.google.com/store/apps/details?id=org.mupen64plusae.v3.fzurita

There are many clones of that one and that is the right one.

**Comment by Jairoon** [comment]
Lla loice pero sigue pasando me quedo en o se sale en el templo de las sombras

**Reply by fzurita** [reply]
Estas ocupando algún cheats? A veces los cheats causan eso.

**Comment by Jairoon** [comment]
No

**Comment by fzurita** [comment]
Ok, selecciona el juego, anda a opciones y prueba cambiando el perfil de emulación a GLideN64-Medium.

**Comment by Jairoon** [comment]
Ya lo ise esta mas omenos lento pero si pasa algo te aviso grasias por su
atension
